src/node/stage/approved.rs

```diff
@@ -782,6 +836,66 @@ impl Approved {
         }
     }
 
+    // TODO: carry out accumulation
```

Comment:
What does this mean?

Reply:
here we trust the first guy give us this info.
It actually shall accumulate the incoming notification and trust by group.

Reply:
Hm, so is this the thing we discussed, where the DKG is performed by the new elders, but then has to be sent to the old ones to sign?

Reply:
yes

Reply:
Hm, but that's that mean that `notify_sibling` then isn't only for siblings after all? Say the current elders are `A`, `B`, `C` and `D` and the new ones are `A`, `B`, `E` and `F`. Then first `A`, `B`, `E`, `F` will complete the DKG, then send the result to `C` and `D` who (together with `A` and `B`) will then vote to sign it? If so, then there is no split involved and the term sibling is misleading.

Reply:
Also maybe expand the TODO a bit. Something like: "accumulate at least quorum of these messages and only then proceed to handle the DKG result."

Reply:
How about change `sibling` to `old_elders` ?

Reply:
Yeah, sounds good.